web/ui/mantine-ui/package.json

```diff
@@ -12,62 +12,62 @@
     "test": "vitest"
   },
   "dependencies": {
-    "@codemirror/autocomplete": "^6.18.4",
```

Comment:
usual the deps bumps is performed before the release branch cut.

Reply:
actually I was expecting this PR to be against `release-3.4` but the branch isn't available yet, maybe I should wait until the cut is performed

Reply:
https://github.com/prometheus/prometheus/pull/16493